pyproject.toml

```diff
@@ -64,7 +64,6 @@ ignore = [
     "E741",
     "EXE001",
     "F405",
-    "F841",
```

Comment:
I suppose this is really the line which applies to the PR title, right? Seems like everything else removes more unused vars?

Reply:
Right, the previous two commits remove all the failures in `torch/` and `test/` and this does "all the other directories", they being

* `.github/'
* `benchmarks/`
* `functorch/`
* `tools/`
* `torchgen/`

I had thought even of splitting this commit into two: one would be just the source code changes, and the other one being the single line change turning on checking for unused variables, in case there's a revert.

I don't want to spam everyone with commits, but it's the most solid way of making the change now I think about it - what do you think?